kwmserver/wrapper.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+if [ -n "$oidc_issuer_identifier" ]; then
+	set -- "$@" --iss="https://$oidc_issuer_identifier"
```

Comment:
This looks wrong. A proper OIDC issuer identifer includes "https://" since the protocol it is part of the identifier.

Reply:
indeed. Before I always prefixed https to the fqdn in the compose file. I have realised this for this case now as well.